src/content/1.6/code/agda/full.agda

```diff
@@ -0,0 +1,41 @@
+open import Data.Unit using (⊤; tt)
+open import Data.Bool using (Bool; false; true; _∧_)
+open import Data.String using (String)
+open import Data.Integer using (ℤ)
+open import Data.Product using (_×_; _,_)
+open import Data.List
+open import Relation.Binary.PropositionalEquality
+
+variable a b c : Set
+
+swap : (a × b) → (b × a)
+swap (x , y) = y , x
+
+alpha : ((a × b) × c) → (a × (b × c))
+alpha ((x , y) , z) = x , (y , z)
```

Comment:
@williamdemeo I on purpose left brackets in here in `x , (y , z)`  I think we are not trying to show that Agda is reasonable Haskell but rather show differences e.g in syntax (or even in approach and what is possible).

I was changing this back and forth couple of times :) WDYT?

Reply:
I think it looks good as it is now.  👍🏼